cmd/operator/README.md

```diff
@@ -31,11 +31,10 @@ environment variables, such as `PROJECT_ID` but these are configured
 automatically if they are not set. The command will give you the uploaded image
 URL and update all necessary configurations to use it.
 
-Next, apply the Kubernetes configuration files, starting with the CRDs:
+Next, install the Kubernetes configuration files using Helm:
```

Comment:
I think we want to still advertise the `kubectl apply -f` option for our manifests to:
1. Avoid being prescriptive
2. Be consistent with the GCP docs

Reply:
Reverted this file to original.

Reply:
Maybe you forgot the commit?

Reply:
LOL, I pushed it into #819. I'm just going to merge that into this branch. I'm having a bad Git day.